Packs/Akamai_SIEM/README.md

```diff
@@ -0,0 +1,46 @@
+<~XSIAM>
+
+## Overview
+
+This pack enables using a Web Application Firewall (WAF), a security filter that protects against HTTP-based attacks by inspecting traffic before it reaches your application.
+Note:
```

Comment:
Pay Attention in bold marking.
And as a new line, checking with the view in Github its all one paragraph.